src/graph.ts

```diff
@@ -462,6 +582,110 @@ export class FactoryGraph {
         return new Set(Array.from(this.containers).filter((node) => node.item === item))
     }
 
+    /**
+     * Return the set of all transfer containers holding one or more of the
+     * given items and no other items
+     * @param items Items for which to find the TransferContainers
+     */
+    getTransferContainers(items: Set<Item>): Set<TransferContainerNode> {
+        let transferContainers = Array.from(this.transferContainers)
+        // Filter only those containing one or more of items
+        transferContainers = transferContainers.filter((node) =>
+            node.items.some((item) => Array.from(items).includes(item)),
+        )
+        // Filter out those containing anything not in items
+        transferContainers = transferContainers.filter((node) =>
+            Array.from(items).some((item) => !node.items.includes(item)),
+        )
+        return new Set(transferContainers)
+    }
+
+    /**
+     * Add transfer units and containers to handle industries that require
+     * >7 incoming links
+     */
+    handleIndustryLinks() {
```

Comment:
Same as in the other review. I would like this to be a standalone function which receives the factory as input.

Reply:
Fix in https://github.com/lgfrbcsgo/du-factory-generator/commit/526633908d47ce18a9744e1edc91bfd42bc579a8